getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -21,11 +23,11 @@ In the import popup, enter the full path to the freshly created directory
 open a file browser and navigate to the ``project.godot`` file the folder
 contains.
 
-.. image:: img/01.game_setup/02.browse_to_project_folder.webp
+.. image:: img/01.game_setup/02.browse_to_project_folder.png
```

Comment:
```suggestion
.. image:: img/01.game_setup/02.browse_to_project_folder.webp
```
This is incorrect, the file is `webp`

Reply:
Half of the files were PNG, I've converted them all to WEBP.